plugins/labhub.py

```diff
@@ -1,7 +1,106 @@
+                     'actual problems. :)',
+        'developers': 'Wow @{}, you are a part of developers team now! :tada: '
+                      'Welcome to our community!',
+        'maintainers': '@{} you seem to be awesome! You are now a maintainer! '
+                       ':tada: Please go through '
+                       'https://github.com/coala/coala/wiki/Membership'
+    }
+
+    def activate(self):
+        super().activate()
+        self.GH = LabHub.get_gh_obj(os.environ.get('GH_TOKEN'))
+        self.ORG = LabHub.get_org(self.GH, 'coala')
+        self.TEAMS = LabHub.get_teams(self.ORG)
+
+    @staticmethod
+    def get_gh_obj(token):
+        """
+        :param token:   github token
+        :returns:       ``github3.GitHub`` object
+        """
+        return github3.login(token=token)
+
+    @staticmethod
+    def get_org(gh, org_name):
+        """
+        :param gh:          ``github3.GitHub`` object
+        :param org_name:    Name of the organization
+        :returns:           ``github3.orgs.Organization`` object
+        """
+        return gh.organization(org_name)
+
+    @staticmethod
+    def get_teams(org):
+        """
+        :param org: ``github3.orgs.Organization`` object
+        :returns:   Dict containing team names as keys and corrosponding
+                    ``github3.orgs.Team`` objects as values.
+        """
+        teams = dict()
+        for team in org.iter_teams():
+            teams[team.name] = team
+        return teams
+
+    @staticmethod
+    def invite(user, team):
+        """
+        Invite user to team.
+        :param user:    Username
+        :param team:    ``github3.orgs.Team`` object
+        """
+        return team.invite(user)
+
+    @re_botcmd(pattern=r'(?:(?:invite)|(?:inv))\s+(\w+)\s*(?:to)\s+(\w+)')
+    def invite_cmd(self, msg, match):
+        """
+        Invite given user to given team. By default it invites to
+        "coala newcomers" team.
+        """
+        invitee = match.group(1)
+        inviter = msg.frm.nick
+
+        team = 'coala newcomers' if match.group(2) is None else match.group(2)
+
+        self.log.info('{} invited {} to {}'.format(inviter, invitee, team))
+
+        if self.TEAMS['coala maintainers'].is_member(invitee):
+            valid_teams = ['newcomers', 'developers', 'maintainers']
+            if team.lower() not in valid_teams:
+                return 'Please select from one of the ' + ', '.join(valid_teams)
+            team_mapping = {
+                'newcomers': 'coala newcomers',
+                'developers': 'coala developers',
+                'maintainers': 'coala maintainers'
+            }
+
+            # Call the helper method after all the validation is done
+            LabHub.invite(invitee, self.TEAMS[team_mapping[team.lower()]])
+            return self.INVITE_SUCCESS[team.lower()].format(invitee)
+        else:
+            return 'Nice try :poop:'
```

Comment:
Better to say you are not a maintainer and hence you don't have the rights to invite. Nice try 💩 .